ChangeLog

```diff
@@ -19,6 +19,8 @@ Bugfix
      in X.509 module. Fixes #2212.
    * Reduce stack usage of `mpi_write_hlp()` by eliminating recursion.
      Fixes #2190.
+   * Assure that required seed length does not exceed MBEDTLS_ENTROPY_BLOCK_SIZE,
```

Comment:
This is not what we do. In the code we are making sure that seeding CTR DRBG and HMAC DRBG is possible even if the randomness source can't return more than MBEDTLS_ENTROPY_BLOCK_SIZE at a time. Given that this is a requirement we want to lift in the near future, I suggest we focus on the actual issue fixed and don't go into details. I was thinking something like this:

> Fix issue in seeding HMAC DRBG when MBEDTLS_SHA512_C is disabled. Reported by dbedev. Fixes #770.